client/src/components/categorical/category.js

```diff
@@ -238,58 +238,66 @@ class Category extends React.Component {
               {""}
             </label>
             {/* Dialog uses portal, can be anywhere/factored out */}
-            <Dialog
-              icon="tag"
-              title="Edit category name"
-              isOpen={
-                annotations.isEditingCategoryName &&
-                annotations.categoryEditable === metadataField
-              }
-              onClose={this.disableEditCategoryMode}
-            >
-              <div className={Classes.DIALOG_BODY}>
-                <div style={{ marginBottom: 20 }}>
-                  <p>New, unique category name:</p>
-                  <InputGroup
-                    autoFocus
-                    onChange={e =>
-                      this.setState({ newCategoryText: e.target.value })
-                    }
-                    leftIcon="tag"
-                  />
-                </div>
-              </div>
-              <div className={Classes.DIALOG_FOOTER}>
-                <div className={Classes.DIALOG_FOOTER_ACTIONS}>
-                  <Tooltip content="Close this dialog without editing the category.">
-                    <Button onClick={this.disableEditCategoryMode}>
-                      Cancel
-                    </Button>
-                  </Tooltip>
-                  <Button
-                    disabled={newCategoryText.length === 0}
-                    onClick={this.handleEditCategory}
-                    intent="primary"
-                  >
-                    Edit category name
-                  </Button>
-                </div>
-              </div>
-            </Dialog>
+
             <span
               data-testid={`category-expand-${metadataField}`}
               style={{
                 cursor: "pointer",
                 display: "inline-block"
               }}
               onClick={() => {
-                this.setState({ isExpanded: !isExpanded });
+                const editingCategory =
```

Comment:
What prevents the user from trying to put two category names into edit mode?    A quick skim of the code makes it look possible -- `activateCategoryEditMode()` in `category.js` doesn't seem to check for this.

Reply:
which leads to the question: does label editing state have the same issue?

Reply:
Let's open a new issue for validation. Issue #930 just concerned changing the form of the input.

Reply:
OK w/ me.  But please file it and cross-link to this PR so we don't lose track of it.